libs/partners/mongodb/langchain_mongodb/vectorstores.py

```diff
@@ -31,8 +32,6 @@
 
 logger = logging.getLogger(__name__)
 
-DEFAULT_INSERT_BATCH_SIZE = 100
```

Comment:
May be better to keep this but change it to 100,000 which is MongoDB's maxWriteBatchSize.